### ISSUE TITLE
Repo for Copy Errors?

### ISSUE THREAD

**Issue by DestinyRoseMurphy** [shiffman/LearningProcessing#229]
Do you have a repo for errors in Learning Processing that are not in the examples of the book? Figure 1-11 stands out: In the third graph, you have points (8,7) and (5,5) both labeled as (3,3). 
![img_0152](https://user-images.githubusercontent.com/29611791/51767448-0ceb0e00-20a3-11e9-82da-77770e9cef2e.JPG)

**Comment by shiffman** [comment]
This is the right place! Just to confirm is this the second edition?

**Comment by DestinyRoseMurphy** [comment]
Yes sir! (also, super cool that I'm talking to you - I'm a brand new programmer thank you so much for responding)
